crowbar_framework/app/models/service_object.rb

```diff
@@ -1148,9 +1146,10 @@ def apply_role(role, inst, in_queue)
             # A new node that we did not know before
             unless old_nodes.include?(node_name)
               @logger.debug "add node #{node_name}"
-              pending_node_actions[node_name] = { remove: [], add: [] } if pending_node_actions[node_name].nil?
+              pending_node_actions[node_name] ||= { remove: [], add: [] }
               pending_node_actions[node_name][:add] << role_name
             end
```

Comment:
I do not remember exactly, I think there were no network data saved in the initial chef-client run when executing instcrowbar step